### PR TITLE
 changed navbar background colour in mobile view, fixes #135

### LINKED ISSUE #135: PyCon logo conflicts with current theme colour in mobile menubar
PyCon logo has the same colour accents as of menubar background in mobile view.

Here is screenshot:

screen shot 2018-06-13 at 7 17 37 pm

Suggestion:

screen shot 2018-06-13 at 6 50 40 pm

White background recommended with theme coloured menubar toggle.

### PR DESCRIPTION
**Pull request by dhavalsavalia**

### CODE REVIEW

**Comment by ananyo2012** [comment]
@dhavalsavalia Looks like you have some previous commits. Please remove them. Also can you provide a preview of your changes ?

**Comment by ananyo2012** [comment]
@dhavalsavalia I have recently registered the repo for netlify builds. So you won't have to generate the preview now. Just remove the older commits and make a new PR to enable netlify build.

**Comment by dhavalsavalia** [comment]
@ananyo2012 done.

**Comment by ananyo2012** [comment]
Can you fix the logo alignment in larger screen widths ?
screenshot from 2018-06-25 00-51-50

**Comment by dhavalsavalia** [comment]
@ananyo2012 I have check my local file and it is working totally fine. I don't know what is causing this problem. I have also used difference checker tool if there is something wrong with `html` file or `style.css` but they are exactly the same. No difference between working local files and netlify build.

**Comment by ananyo2012** [comment]
I still see it. Can someone from @pythonindia/pycon-2018-team verify ? 

> UPDATE: I noticed that Speakers and Tickets tabs were added while resolving git merge conflicts, I guess.
No they are required in the navbar. We added them in all pages.

**Comment by ananyo2012** [comment]
Looks like we have this issue in the home page as well. Let me get feedback from other reviewers.

File: custom/css/style.css [hunk 1]
```diff
@@ -1993,4 +1995,4 @@ ul.faq-answer li a:hover{
 .icon:hover {
   opacity:0.5;
   z-index:10;
-}
+}
```
Comment (realslimshanky) [inline custom/css/style.css]: Please add a new line here.

**Comment by dhavalsavalia** [comment]
Sorry, I started college so didn't push any commits.

File: custom/css/style.css (hunk 1 above)
Comment (dhavalsavalia) [reply custom/css/style.css]: Please see: 5223e3a

**Comment by dhavalsavalia** [comment]
@ananyo2012 sorry, I started college so couldn't contribute but I pushed commits with fixes. Please review them.

**Comment by ananyo2012** [comment]
The logo looks bit misaligned in medium screen width. But let's merge this PR now. Thanks for your contribution. Please have a look at other issues as well https://github.com/pythonindia/inpycon2018/issues

**Pull request merged**